File: backend/src/main/java/com/team05/petmeeting/domain/donation/dto/CompleteRes.java
package com.team05.petmeeting.domain.donation.dto;

import com.team05.petmeeting.domain.donation.enums.DonationStatus;
import lombok.Builder;

@Builder
public record CompleteRes (
    Long id,
    int amount,
    DonationStatus status,
    Long campaignId
){
    public static CompleteRes create() {
        return new CompleteRes(1L, 10000, DonationStatus.PENDING, 1L);

    }
}

File: backend/src/main/java/com/team05/petmeeting/domain/donation/dto/CompleteRes.kt
package com.team05.petmeeting.domain.donation.dto

import com.team05.petmeeting.domain.donation.enums.DonationStatus
import lombok.Builder

@Builder
data class CompleteRes(
    val id: Long,
    val amount: Int,
    val status: DonationStatus,
    val campaignId: Long
) {
    companion object {
        @JvmStatic
        fun create(): CompleteRes {
            return CompleteRes(1L, 10000, DonationStatus.PENDING, 1L)
        }
    }
}
